onboarding_auth/onboardingAuthData/src/main/java/com/alexafit/onboardingauthdata/model/remote/Login.kt

```diff
@@ -0,0 +1,6 @@
+package com.alexafit.onboardingauthdata.model.remote
+
+data class Login(
```

Comment:
Whenever im sending data classes to an api, I like to add "Dto" at the end
So in this case id name it
LoginDto